src/ui/RealmBrowser/Content/index.tsx

```diff
@@ -718,8 +747,19 @@ class ContentContainer extends React.Component<
       this.write(() => {
         // Writing makes no sense if the realm was not loaded
         if (realm) {
-          // Adding a primitive value into a list
-          if (isPrimitive(className)) {
+          if (embeddedInfo) {
+            const { parent, key } = embeddedInfo;
+
+            if (focus && focus.kind === 'list') {
+              // Using this method as instanceof Realm.List is not currently valid
```

Comment:
What is ment by "is not currently valid"?

Reply:
This declaration doesn't allow for `instanceof`: https://github.com/realm/realm-js/blob/master/lib/index.d.ts#L311-L313
I'm contemplating re-writing these declarations to abstract classes with private constructors, but haven't yet tested it out.